docs/topics/installing-cli-tool.adoc

```diff
@@ -46,35 +33,143 @@ When you encounter `<{ProductShortName}_HOME>` in this guide, replace it with th
 [id="installing-using-podman_{context}"]
 == Installing the {CLINameTitle} using Podman
 
+You can install the {CLINameTitle} using `podman pull`. This is the recommended approach, except when migrating applications built with a .NET framework on Windows where Docker is required.
```

Comment:
> You can install the {CLINameTitle} using `podman pull`. This is the recommended approach,

Is this correct?  Until now, the recommended approach was to download `.zip` file

Reply:
You're correct. I will fix this.

Reply:
Fixed.